hail/python/hail/docs/change_log.md

```diff
@@ -22,6 +22,23 @@ Please note that **forward compatibility should not be expected, especially
 relating to file formats**: this means that it may not be possible to use
 an earlier version of Hail to read files written in a later version.
 
+## Version 0.2.42
+
+Released 2020-05-27
+
+### New Features
+
+- (hail#8822) Add optional non-centrality parameter to `hl.pchisqtail`.
+- (hail#8861) Add `contig_recoding` option to `hl.experimental.run_combiner`.
+
+### Bug fixes
+
+- (hail#8863) Fixes VCF combiner to successfully import GVCFs with alleles called as <NON_REF>.
+- (hail#8845) Fixed issue where accessing an element of an ndarray in a call to Table.transmute would fail (#8845)
```

Comment:
needs a period

Reply:
also forgot to delete the pr numbe rat the end